meson.build

```diff
@@ -3666,6 +3691,11 @@ endif
 # Let the test runner know whether we're running under asan and export
 # some paths. We use a file on disk so that it is possible to run the
 # test scripts from commandline without setting any variables.
+python_module = import('python')
+python = python_module.find_installation(
+    get_option('python'),
+    required : true,
+    disabler : true)
```

Comment:
Why is this in here?
It seems to work without.

Reply:
Nevermind, I saw why. But IMHO then this code in `libmount/python/` can be dropped.

Reply:
Sure thing, I'll deduplicate it.

Reply:
Done.